README.md

```diff
@@ -1,82 +1,65 @@
+* [Geeks For Geeks: HTML Section](https://www.geeksforgeeks.org/html-tutorials/)
+* [HTML Cheatsheet](https://digital.com/tools/html-cheatsheet/) NEW
+* [HTML Cheat Sheet with Examples](https://htmlreference.io/) NEW
 ---
-
 ## CSS
-
 * [Mozilla Developer Network | CSS Section](https://developer.mozilla.org/en-US/docs/Learn/CSS)
 * [Unfolding the Box Model](https://rupl.github.io/unfold/)
 * [CSS3 Cheatsheet](https://www.onblastblog.com/css3-cheat-sheet/)
-
+* [Cheat Sheet with Examples](https://cssreference.io/) NEW
+* [Master Cheat  Sheets for lots of Langs](https://devhints.io/) NEW
+* [For Creation of  Gradients](https://cssgradient.io/) NEW
 ### Flexbox Games
-
 * [CSS Diner](https://flukeout.github.io/)
 * [Flexbox Defense](http://www.flexboxdefense.com/)
 * [Flexbox Froggy](https://flexboxfroggy.com/)
-
 ### Grid
-
-* [Grid Garden: Game](https://cssgridgarden.com/)
-* [Grid Critters: Game](https://www.gridcritters.com/)
-* [MDN CSS Grid: Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)
-* [Learn CSS Grid: Reference](https://learncssgrid.com/)
-
+ * [Grid Garden: Game](https://cssgridgarden.com/)
+ * [Grid Critters: Game](https://www.gridcritters.com/)
+ * [MDN CSS Grid: Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)
+ * [Learn CSS Grid: Reference](https://learncssgrid.com/)
 ---
-
-## HTML
-
-* [Mozilla Developer Network | HTML Section](https://developer.mozilla.org/en-US/docs/Learn/HTML)
-* [HTML Reference](https://htmlreference.io/)
-* [HTML5Doctor](http://html5doctor.com/)
-* [HTML.com](https://html.com/)
-* [HTML Color Codes](https://html-color-codes.info/)
-* [HTMLDog HTML](https://htmldog.com/guides/html/)
-* [Geeks For Geeks: HTML Section](https://www.geeksforgeeks.org/html-tutorials/)
-* [HTML Cheatsheet](https://digital.com/tools/html-cheatsheet/)
-
+## JavaScript
+* [Modern JavaScript Tutorial](https://javascript.info/)
+* [Mozilla Developer Network | JavaScript Section](https://developer.mozilla.org/en-US/docs/Learn/JavaScript)
+* [Sitepoint: Truthy and Falsy](https://www.sitepoint.com/javascript-truthy-falsy/)
+* [JavaScript for Cats](http://jsforcats.com/)
 ---
-
-## PODCASTS
-
+## Podcasts
 * [Syntax.fm](https://syntax.fm/)
-
 ---
-
-## Developer websites
+## Developer Websites
 * [Dev.to](https://dev.to/)
 * [CSS Tricks](https://css-tricks.com/)
 * [Smashing Magazine](https://www.smashingmagazine.com/)
 * [SitePoint](https://www.sitepoint.com/blog/)
-
 ---
-
-## Developer newsletters
+## Developer Newsletters
 * [JavaScript Weekly](https://javascriptweekly.com/)
 * [NodeJS Weekly](https://nodeweekly.com/)
 * [CSS Weekly](https://css-weekly.com/)
 * [Frontend Mentor](https://frontendmentor.us19.list-manage.com/subscribe?u=efe35252db9bea1dac75aca25&id=8e63782fc5)
 * [Mobile Dev Weekly](https://mobiledevweekly.com/)
-
 ---
-
-## Other
-
+## YouTubers / Other
+* [IFHT - Comedy Music Videos](https://www.youtube.com/channel/UCTs59UCfP4YLUt6pDR\_uLtg) NEW
```

Comment:
This link is broken. If it's music or anything on programming you should just share it in the chat. The reason being that if we open the door to that then we'd have dozens of links on all kinds of topics. Need to keep this list very strict so it's short.

Reply:
Sure delete it out!